docs/helm/README.md

```diff
@@ -1,3 +1,9 @@
+---
+title: Overview
+parent: Deployment
+grand_parent: Helm
+---
```

Comment:
```suggestion
nav_order: 1
---
```

so it appear as first in the navigation menu

Reply:
Shouldn't the docs/index.md be first?